src/Heart.java

```diff
@@ -6,7 +6,7 @@ public void applyCollapsePenalties(ImmuneSystem system) {
     }
 
     public void applyDamage(Pathogen cell) {
-        if (getPlayerControl() < 0) { return; }
+        if (getPercentHealth() < 0) { return; }
```

Comment:
Consider renaming percentHealth to organHealth to avoid confusion with globalHealth or any other health bars.